napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -337,13 +341,79 @@ def setup_ui(self, enabled=True):
         self.source_choice_dropdown.currentTextChanged.connect(
             self._populate_version_dropdown
         )
-        self.row2.addWidget(
-            self.install_info_button,
-            0,
-            7,
-            1,
-            1,
-            alignment=Qt.AlignmentFlag.AlignTop,
+
+        # Information widget for installed packages
+        self.info_widget = QWidget(self)
+        self.info_widget.setLayoutDirection(Qt.LeftToRight)
+        self.info_widget.setObjectName("info_widget")
+        self.info_widget.setFixedWidth(180)
```

Comment:
Fixes

2. Align installation info, Make a standard size? ( I know this button adjusts by the sizes on the drop down menus held within. Eliding may help here (see item below).